src/bin/command.ts

```diff
@@ -0,0 +1,200 @@
+#!/usr/bin/env node
```

Comment:
moving the hasang seems like an accident, as `bin/release-please.ts` is still the entrypoint?

Reply:
Good catch - yeah `bin/release-please.ts` is still the entry point and this is just the `yargs` configuration. I'll update this.